server/liveblog/client_modules/client_modules.py

```diff
@@ -86,6 +89,57 @@ class ClientPostsService(PostsService):
     pass
 
 
+class ClientCollectionsResource(CollectionsResource):
+    datasource = {
+        'source': 'collections',
+        'default_sort': [('name', 1)]
```

Comment:
Is the reason for this resource just to provide a shortcut for a request with a certain kind of sort? In which case, it's a lot of code to save having to add a parameter on the front end.